.github/templates/metagraph_data_l1/initial_validator/action.yml

```diff
@@ -0,0 +1,83 @@
+name: Metagraph L1 - Data Initial Validator
+
+inputs:
+  METAGRAPH_ID:
+    required: true
+  CL_PUBLIC_HTTP_PORT:
+    required: true
+  CL_P2P_HTTP_PORT:
+    required: true
+  CL_CLI_HTTP_PORT:
+    required: true
+  CL_GLOBAL_L0_PEER_HTTP_PORT:
+    required: true
+  CL_L0_PEER_HTTP_PORT:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create directories
+      shell: bash
+      run: |
+        cd .github/config
+        mkdir -p containers/
+        cd containers
+        mkdir data-l1
+        mkdir -p jars
+
+    - name: Generate Metagraph L1 - Data JAR
+      shell: bash
+      run: |
+        cd .github/templates/project_template/
+        sbt dataL1/assembly
+
+    - name: Move Metagraph L1 - Data JAR
+      shell: bash
+      run: |
+        mv .github/templates/project_template/modules/data_l1/target/scala-2.13/project_template-data_l1-assembly-* .github/config/containers/jars/data-l1.jar
+
+    - name: Running Metagraph L1 - Data Initial Validator
+      shell: bash
+      run: |
+        export CL_KEYSTORE=token-key.p12
+        export CL_KEYALIAS=token-key
+        export CL_PASSWORD=password
+        
+        cd .github/config/containers/global-l0/genesis
+        export GLOBAL_L0_NODE_ID=$(java -jar ../../jars/cl-wallet.jar show-id)
+
+        cd ../../metagraph-l0/genesis
+        export CURRENCY_L0_GENESIS_NODE_ID=$(java -jar ../../jars/cl-wallet.jar show-id)
+        export METAGRAPH_IDENTIFIER=$METAGRAPH_ID
+
+        cd ../../data-l1
+        mkdir initial-validator
+        cd initial-validator
+
+        java -jar ../../jars/cl-keytool.jar generate
+        
+        export CURRENCY_L1_INITIAL_VALIDATOR_NODE_ID=$(java -jar ../../jars/cl-wallet.jar show-id)
+
+        export CL_PUBLIC_HTTP_PORT=${{ inputs.CL_PUBLIC_HTTP_PORT }}
+        export CL_P2P_HTTP_PORT=${{ inputs.CL_P2P_HTTP_PORT }}
+        export CL_CLI_HTTP_PORT=${{ inputs.CL_CLI_HTTP_PORT }}
+        export CL_APP_ENV=dev
+        export CL_ENV=dev
```

Comment:
just `CL_APP_ENV`. the `CL_ENV` is not detected by the tessellation

Reply:
Fixed